packages/wonder-blocks-link/src/components/__docs__/link.stories.js

```diff
@@ -42,30 +47,32 @@ Default.args = {
     children: "Hello, world!",
 };
 
-export const Basic: StoryComponentType = () => (
+export const Primary: StoryComponentType = () => (
     <Link href="#">The quick brown fox jumps over the lazy dog.</Link>
 );
 
-Basic.parameters = {
+Primary.parameters = {
     docs: {
         storyDescription: `Minimal link usage.
             This links to the top of the page.`,
     },
 };
 
-Basic.play = async ({canvasElement}) => {
+Primary.play = async ({canvasElement}) => {
     const canvas = within(canvasElement);
 
     const link = canvas.getByRole("link");
 
+    await expect(link).toHaveStyle(`color: ${Color.blue}`);
```

Comment:
I added the initial color check to all the interaction tests because I ran into an issue at one point where the initial color was wrong and it wasn't being caught anywhere.